examples/pcf8574_i2c/main.cpp

```
@@ -0,0 +1,35 @@
+    ITEM_BASIC("Settings"),
+    ITEM_BASIC("Blink SOS"),
+    ITEM_BASIC("Blink random"));
+// clang-format on
+
+LCD_I2C lcd(0x27, 16, 2);
+LCD_I2CAdapter lcdAdapter(&lcd);
+CharacterDisplayRenderer renderer(&lcdAdapter, 20, 4, 0xDF, 0xE1);
+LcdMenu menu(renderer);
+KeyboardAdapter keyboard(&menu, &Serial);
+
+void setup() {
```

cppcheck (2.10-2)

[style] The function 'setup' is never used.

(unusedFunction)